backend/btrixcloud/operator.py

```diff
@@ -461,6 +469,8 @@ async def sync_crawl_state(self, redis_url, crawl, status, pods):
                 allowed_from=["starting", "running"],
             )
 
+            # resync after 3 seconds
+            status.resyncAfter = 3
```

Comment:
Looks good - only question is whether we should use a constant here, or make it configurable?

Reply:
Refactored to make configurable!